subsys/mpsl/init/mpsl_init.c

```diff
@@ -20,6 +20,9 @@
 #if defined(CONFIG_MPSL_TRIGGER_IPC_TASK_ON_RTC_START)
 #include <hal/nrf_ipc.h>
 #endif
+#if IS_ENABLED(CONFIG_SOC_PLATFORM_HALTIUM) || IS_ENABLED(CONFIG_SOC_PLATFORM_NRF54L)
+#include <nrfx_grtc.h>
```

Comment:
Why do we need to include nrfx_grtc here?

Reply:
I've checked it again and it builds without it. Lines were removed.